onyo/commands/get.py

```diff
@@ -51,9 +51,10 @@
         type=str,
         default=None,
         help="""
-            Criteria to match assets in the form ``KEY=VALUE``, where VALUE is a
-            python regular expression. Pseudo-keys such as ``path`` can also be
-            used. Special values supported are:
+            Criteria to match assets in the form ``KEY=VALUE``, where **VALUE**
```

Comment:
> ``KEY=VALUE``, where **VALUE**

I am never sure about the english comma rules. Is that one here correct?

Reply:
Yes, this looks correct to me.